src/component/feature/FeatureView2/FeatureVariants/FeatureVariantsList/AddFeatureVariant/AddFeatureVariant.tsx

```diff
@@ -204,6 +208,17 @@ const AddVariant = ({
         ]);
     };
 
+    const isJSON = jsonString => {
+        try {
+            let parsedString = JSON.parse(jsonString);
+            if (parsedString && typeof parsedString === 'object') {
+                return true;
+            }
+        } catch (e) {}
```

Comment:
I think we should handle this more gracefully. Perhaps replace the catch with a finally clause and return false?

```
try { 
  ... 
} finally {
   return false
}
```